src/utils/util.js

```diff
@@ -20,4 +21,6 @@ export function downloadAsFile(content, name, type = 'text/csv') {
   a.remove();
 }
 
-export default isValidAddress;
+export function nanolikeToLIKE(nanolike) {
+  return Number(new BigNumber(nanolike).dividedBy(1e9).toFixed(0));
```

Comment:
Prefer `shiftedBy` instead of `dividedBy`
Also why not just use `toNumber()` instead of `toFixed(0)`?

Reply:
Directly using `toNumber()` will leave annoying digital points?